cli/cmd/root.go

```diff
@@ -64,6 +64,6 @@ func setLogLevel(logLevel string) {
 	case "panic":
 		logrus.SetLevel(logrus.PanicLevel)
 	default:
-		logrus.Fatal("Unrecognized log level entry: %s", logLevel)
+		logrus.Fatalf("Unrecognized log level entry:", logLevel)
```

Comment:
`logrus.Fatalf("Unrecognized log level entry: %s", logLevel)` and it should be g2g, would have just fixed online but think that would bork our gpg signature requirement.